AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/LocationsRoutes.java

```diff
@@ -75,6 +75,11 @@ public Location updateEntity(Request req, Response res, int id) {
     }
 
     private void validateLocationModel(Location location) {
+
+        if (location.getSiteName().length() > 20) {
```

Comment:
This will error if the sitename is null - do we need to care about that?

Reply:
(Given the context, and that location sitename is meant to be non-null, I'm happy to assume this is reasonable)